#avoid.action
#goal
bool avoid_enable#启动抓取
---
#result
bool avoid_success#抓取成功
---
#feedback
uint8 avoid_current_step#当前步骤
